packagers/pnpm/prisma/schema.prisma

```diff
@@ -0,0 +1,14 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("PACKAGERS_NPM_PG_URL")
```

Comment:
I see that each package manager has it's own db. Should we set that up for `pnpm`?

Reply:
Yes, that would be necessary - technically and optimally even one for each test. That is a decision we made in the beginning that is not scaling great, but we did not have the time to fix it yet.

Easiest way to get the databases right now is to use the internal database provision tool that @madebysid posted in #general this week, then set the connection strings in the repo secrets (if you can't do that, just DM them to me and I can do that).